tests/unit/helpers.js

```diff
@@ -182,10 +182,17 @@ export function fake<T>(original: Object, methods: Object = {}): T {
  * Returns a fake Firefox client as would be returned by
  * connect() of 'node-firefox-connect'
  */
+
+type FakeFirefoxClientParams = {
+  requestResult?: Object,
+  requestError?: Object | void,
+  makeRequestResult?: Object,
+  makeRequestError?: Object | void,
```

Comment:
why do you need `void` here? I think these were just left over from before you removed null (which was the right move)